frontend/src/routes/trade/Review.svelte

```diff
@@ -0,0 +1,130 @@
+<script lang="ts">
+    import NewsTile from "$components/news/NewsTile.svelte";
+    import api from "$lib/api";
+    import type { Article } from "$lib/api/news";
+    import type { MakeTrade } from "$lib/api/trades";
+    import { ApiError } from "$lib/request";
+    import { goto } from "$app/navigation";
+    import { browser } from "$app/environment";
+    import { stepUrl } from "./Steps.svelte";
+
+    export let initialState: MakeTrade;
+
+    $: ({
+        side,
+        ticker,
+        text_evidence: textEvidence,
+        amount,
+        article_evidence: articleEvidence,
+        type,
+    } = initialState);
+
+    let articlesRequest = api.pendingRequest<Article[]>();
+    $: if (browser) articlesRequest = api.news().articles(articleEvidence);
+
+    let errorMessage = "";
+
+    let makingTrade = false;
+    async function makeTrade() {
+        errorMessage = "";
+        makingTrade = true;
+        try {
+            await api.trades().make(initialState);
+        } catch (e) {
+            makingTrade = false;
+            if (e instanceof ApiError) {
+                errorMessage = e.message;
+            } else {
+                errorMessage = "Something went wrong!";
+            }
+            return;
+        }
+
+        makingTrade = false;
+        await goto(stepUrl(5, initialState));
+    }
+</script>
+
+<div class="block">
+    <div class="columns">
+        <div class="column" />
+        <div class="column is-two-thirds">
+            <h2 class="title is-2">Order Summary</h2>
+            <br />
+            <div class="box has-background-grey-dark">
+                <nav class="level">
```

Comment:
Slightly prefer `Ticker | Side | Amount | Type` to echo the actual sequence of things in the process

Reply:
Sure, the only reason I did it the other way was because you can read it directly, e.g. `BUY 34 Units (of) APPL`, but will change